portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java

```diff
@@ -0,0 +1,76 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.gson.Gson;
+import com.google.sps.data.Login;
+import java.io.IOException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles user logins */
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    boolean loginStatus;
+    String userEmail;
+    String url;
+
+    // Gets reference to a UserService
+    UserService userService = UserServiceFactory.getUserService();
+
+    // Checks and modify behavior based on status of user log in 
+    if (userService.isUserLoggedIn()) {
+      // Set login status to true
```

Comment:
I've noticed in your PRs that you're very detailed with comments. That's a great habit and you should keep it up. One suggestion I'd make though is to limit the number of comments that only repeat what the code says

// Set login status to true
loginStatus = true

is redundant because the comment only communicates the same information as the code. Comments are really useful when they provide additional context that a reader may not have. Take a pass over this PR and see if there are any comments that can be removed as redundant.